docs/crr-to-aws-s3.md

```diff
@@ -75,3 +75,334 @@ env_replication_endpoints:
+    ```
+
+5. Store the credentials generated in step 3 as the profile `<source-profile>`:
+
+    ```
+    mkdir ~/.aws
+    vi ~/.aws/credentials
+    ```
+
+    The completed file should be similar to the following:
+
+    ```
+    [<source-profile>]
+    aws_access_key_id = <access-key>
+    aws_secret_access_key = <secret-key>
+    ```
+
+6. Set up replication on your buckets:
+
+    ```
+    node ~/replication/backbeat/bin/setupReplication.js setup \
+    --source-bucket <source-bucket> \
+    --source-profile <source-profile> \
+    --target-bucket <target-bucket> \
+    --site-name <target-site-name>
+    ```
+
+### Detailed Example
+
+1. Create source and target buckets:
+
+    ```
+    aws s3api create-bucket \
+    --bucket <source-bucket> \
+    --endpoint http://<source> \
+    --profile <source-profile>
+    ```
+
+    ```
+    aws s3api create-bucket \
+    --bucket <target-bucket> \
+    --profile <target-profile>
+    ```
+
+2. Enable bucket versioning on each bucket:
+
+    ```
+    aws s3api put-bucket-versioning \
+    --bucket <source-bucket> \
+    --versioning-configuration Status=Enabled \
+    --endpoint http://<source> \
+    --profile <source-profile>
+    ```
+
+    ```
+    aws s3api put-bucket-versioning \
+    --bucket <target-bucket> \
+    --versioning-configuration Status=Enabled \
+    --profile <target-profile>
+    ```
+
+    Verify that bucket versioning is enabled on each bucket:
+
+    ```
+    aws s3api get-bucket-versioning \
+    --bucket <source-bucket> \
+    --endpoint http://<source> \
+    --profile <source-profile>
+    ```
+
+    ```
+    aws s3api get-bucket-versioning \
+    --bucket <target-bucket> \
+    --profile <target-profile>
+    ```
+
+3. Establish a bucket policy to allow the Backbeat replication engine to
+    retrieve the appropriate information. Create the file _policy.json_ with the
+    following content:
+
+    ```
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "s3:GetObjectVersion",
+                    "s3:GetObjectVersionAcl",
+                    "s3:GetObjectVersionTagging"
+                ],
+                "Resource": [
+                    "arn:aws:s3:::bennett-scality-doc-test/*"
+                ]
+            },
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "s3:ListBucket",
+                    "s3:GetReplicationConfiguration"
+                ],
+                "Resource": [
+                    "arn:aws:s3:::bennett-scality-doc-test"
+                ]
+            },
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "s3:ReplicateObject",
+                    "s3:ReplicateDelete",
+                    "s3:ReplicateTags"
+                ],
+                "Resource": "arn:aws:s3:::bennett-federationtest/*"
```

Comment:
For an example AWS bucket name, maybe use a more self-explanatory name like `bennett-aws-bucket`? Or stick with the generic `<target-bucket>`. Maybe same applies above for the source bucket.

Reply:
Yes, this is meant to be `<target-bucket>`! That is left over from when I was testing :smile:.